package/cfg/rke2-cis-1.23-hardened/node.yaml

```diff
@@ -134,18 +134,18 @@ groups:
         scored: true
 
       - id: 4.1.9
-        text: "Ensure that the kubelet --config configuration file has permissions set to 644 or more restrictive (Automated)"
+        text: "Ensure that the kubelet --config configuration file has permissions set to 600 or more restrictive (Automated)"
```

Comment:
Does CIS 1.23 benchmark guide ask for 600 or 644?

Reply:
well its actually interesting it check for 644, but it fails because we set it to 600, which is more restrictive it just the comparison is wrong

Reply:
yeah 600 is stricter anyway, so this is ok